rtengine/procevents.h

```diff
@@ -24,7 +24,7 @@ namespace rtengine
 // Aligned so the first entry starts on line 30
 enum ProcEventCode {
     EvPhotoLoaded = 0,
```

Comment:
While you're revising this file, could you add a few blank lines to bump this one down to line 30 like the comment says?

There's also line 203 in refreshmap.cc which should be removed to preserve the alignment.

Reply:
Hmm, before I do that, we should figure out *why* it starts on line 30.  Perhaps someone attempted to do an auto-parser script as a companion to generateUnusedKeys, similar to my findorphans?

No such script currently exists (and I see a decent amount of evidence it never did...), it may be better to simply delete the comment and have a script that can handle the start line moving around and/or commented lines.

As to line 203, that needs some investigation.  It looks like EvCATEnabled was moved from ALLNORAW to LUMINANCECURVE - looks like @Desmis did that in 5abd92209 - so that appears to be recent and intentional, @Desmis  is there a reason the old value was left in there but commented, as opposed to just changing it?

Reply:
> we should figure out why it starts on line 30

I did this long time ago to make it easier to (manually) check for constency between procevents.h and refreshmap.cc
For examplae get an entry in procevents.h like `EvClip = 12,`. If the corresponding entry in refreshmap.cc is not in line 30+12 = 42, something is wrong.

Reply:
Longterm we might want to figure out a way to automate that a bit.  I'll work on fixing the alignment for now as libclang's comments support is...  Hard to understand to say the least so automating that task is probably a post-5.9 thing.

Thanks for the information!

Reply:
So I just looked at this - the comment is on line 28, but the first entry already is on line 30, no action needed there.

Reply:
The one in refreshmap.cc starts on 30 but this one is on line 26.

Reply:
Aha, missed that.

Fixed.  I amended the last commit and forcepushed since this one was so small.